utilities/is-05-control/README.md

```diff
@@ -29,29 +29,38 @@ optional arguments:
   --version VERSION     Version of IS-05 API of DuT
   -s, --sender          Configure NMOS Sender
   -r, --receiver        Configure NMOS Receiver
-  --request REQUEST     JSON data to be sent in the request to configure
-                        sender
-  --sdp SDP             SDP file to be sent in the request to configure
-                        receiver
+  --request REQUEST     JSON data to be sent in the request to configure sender
+  --sdp SDP             SDP file to be queried from a Sender (write) or be sent to a Receiver (read)
   -u UUID, --uuid UUID  UUID of resource to be configured
 ```
 
-Example call to change receiver
+Inside the script the IS-05 device can be controlled with the following keys
 ```
-python3 is05Control.py --ip <hostname or IP> --port <IS-05 Port> --version <IS-05 Version> --receiver --uuid <Receiver ID> --request tune-receiver-to-reference.json --sdp video-1080i-7.sdp
+Press 'e' to set master_enable True
+Press 'd' to set master_enable False
+Press 'c' to set a valid config on a Sender (JSON) or a Receiver(SDP)
+Press 'u' to set a dummy config on a Sender (JSON) or a Receiver(SDP)
+Press '7' to set 2022-7 Sender to dummy config
+Press 's' to get SDP file (and save to "./new.json" from a Sender)
```

Comment:
Why does it say save to _./new.json_? That's a weird file extension for SDP! Ah, it's actually the name used in the `--sdp` arg or _latest.sdp_ by default? I guess when you grabbed this text from a run you had passed `--sdp new.json`?! How about saying `"./<SDP>"` or something here?

Another thought... Why is saving only supported for Senders? Hmm, I suppose because you don't want to overwrite an SDP file provided via `--sdp` on the command line?

Reply:
new.json is indeed a mistake.

Right,  It makes sense to save from a Sender but just display the patched SDP from a receiver.